manifests/kustomize/env/aws/README.md

```diff
@@ -0,0 +1,82 @@
+# Sample installation
+
+1. Create an EKS cluster and setup kubectl context
+
+Using configuration file to simplify EKS cluster creation process:
+```
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: kfworkshop
+  region: us-west-2
+  version: '1.13'
```

Comment:
Let's change to 1.17 if you determine to give cluster creation instructions.

Reply:
Will fix